packages/perseus-core/src/utils/util.ts

```diff
@@ -0,0 +1,76 @@
+/* Note(tamara): Brought over from the perseus package packages/perseus/src/util.ts file.
+May be useful to bring other perseus package utilities here. */
```

Comment:
The functions here are still used in other widgets as well, specifically categorizer, radio, and sorter, so I figured I should leave them here for now.